include/psa/crypto.h

```diff
@@ -186,11 +184,11 @@ psa_status_t psa_crypto_init(void);
  *    psa_generate_derived_key() or psa_copy_key(). This function reads
  *    the attribute structure, creates a key with these attributes, and
  *    outputs a handle to the newly created key.
- * -# The attribute structure is now no longer necessary. If you called
- *    psa_set_key_domain_parameters() earlier, you must call
- *    psa_reset_key_attributes() to free any resources used by the
- *    domain parameters. Otherwise calling psa_reset_key_attributes()
- *    is optional.
+ * -# The attribute structure is now no longer necessary.
+ *    You may call psa_reset_key_attributes(), although this is optional
```

Comment:
Shouldn't we remove `psa_reset_key_attributes()` from the API too?

Reply:
No. It stays. You can get away without it now, but it's extremely likely that a future API version will require it. You are supposed to call it after `psa_get_key_attributes`.

Reply:
Do you say that it is extremely likely, because of the reintroduction of `key_attributes` or do you have some other use case in mind as well?

Reply:
Domain parameters are the definite thing I have in mind. But also I'd be surprised if nothing else came up, especially when we consider more server and other rich platform use cases.